vita3k/host/src/pkg.cpp

```diff
@@ -0,0 +1,323 @@
+// Vita3K emulator project
+// Copyright (C) 2020 Vita3K team
+//
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation; either version 2 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along
+// with this program; if not, write to the Free Software Foundation, Inc.,
+// 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+#include <PsvPfsParserConfig.h>
+#include <Utils.h>
+#include <app/functions.h>
+#include <crypto/aes.h>
+#include <host/functions.h>
+#include <host/pkg.h>
+#include <host/sce_types.h>
+#include <host/sfo.h>
+#include <host/state.h>
+#include <io/device.h>
+#include <rif2zrif.h>
+#include <util/bytes.h>
+#include <util/log.h>
```

Comment:
it would be smart to sort this I think
```
// section for host/* headers
#include <host/x>
#include <host/yz>

// section for util/* headers
#include <util/abc>
#include <util/defgh>

// section for psvfstools headers
#include <blah blah bla>
```

it's a lot to look at all together